pkg/apis/kops/v1alpha2/componentconfig.go

```diff
@@ -205,6 +205,8 @@ type KubeletConfigSpec struct {
 	ProtectKernelDefaults *bool `json:"protectKernelDefaults,omitempty" flag:"protect-kernel-defaults"`
 	// CgroupDriver allows the explicit setting of the kubelet cgroup driver. If omitted, defaults to cgroupfs.
 	CgroupDriver string `json:"cgroupDriver,omitempty" flag:"cgroup-driver"`
+	// HousekeepingInterval allows to specify interval between container housekeepings.
+	HousekeepingInterval string `json:"housekeepingInterval,omitempty" flag:"housekeeping-interval"`
```

Comment:
```suggestion
	HousekeepingInterval *metav1.Duration `json:"housekeepingInterval,omitempty" flag:"housekeeping-interval"`
```

Reply:
Thanks for the review. I've updated it.

Reply:
Happy to do it. Please also run `make apimachinery` and `make crds` to update the output.